test/run-drun/max-stack-variant.mo

```diff
@@ -2,26 +2,15 @@
 import { errorMessage; performanceCounter; rts_heap_size; rts_max_stack_size; debugPrint; } = "mo:⛔";
```

Comment:
No biggie, but why did you change the test output? I don't think it would have changed output (but maybe it did in previous version).

Reply:
I observed a lower limit for the stack with the incremental GC (length `31_744` instead of `35_840`). For the other GCs, it is still the same. Probably because of more stack space with the extra logic and temporary state for the write barrier. As the results differ between the GC versions, I reduced the output.

Reply:
Update: I analyzed it further: It is the **allocation barrier** call that consumes the extra stack space (the call to the RTS), and reduces the limit for the incremental GC.